tools/validate_taipy_install.py

```diff
@@ -0,0 +1,52 @@
+import logging
+import os
+import sys
+
+
+def test_import_taipy_packages() -> bool:
+    """
+    Import taipy package and call gui, Scenario and rest attributes.
+    """
+    import taipy as tp
+
+    valid_install = True
+    if not hasattr(tp, "gui"):
+        logging.error("Taipy installation has no attribute gui")
+        valid_install = False
+    if not hasattr(tp, "Scenario"):
+        logging.error("Taipy installation has no attribute Scenario")
+        valid_install = False
+    if not hasattr(tp, "rest"):
+        logging.error("Taipy installation has no attribute rest")
+        valid_install = False
+
+    return valid_install
+
+
+def is_taipy_gui_install_valid() -> bool:
+    from pathlib import Path
+
+    import taipy
+
+    taipy_gui_core_path = Path(taipy.__file__).absolute().parent / "gui_core" / "lib" / "taipy-gui-core.js"
+    if not taipy_gui_core_path.exists():
+        logging.error("File taipy-gui-core.js not found in taipy installation path")
+        return False
+
+    taipy_gui_webapp_path = Path(taipy.__file__).absolute().parent / "gui" / "webapp"
+    if not any(fname.endswith('.js') for fname in os.listdir(taipy_gui_webapp_path)):
```

Comment:
Looks good to me 
what's the behavior of os.listdir if the path does not exist ?
Should the existence of the webapp dir be tested first ?

Reply:
Good point, I've added a check for that.